### ISSUE TITLE
Inconsiderate App Store reviews need follow-up?

### ISSUE THREAD

**Issue by Smiles2go** [hape42/dailygammon#132]
Version 2.0 (680) from 08. July 2024 14:28:22

This bug almost feels like special case of "You can't patch stupid".

A few App Store user feedback messages for DG give low ratings for mostly invalid reasons. You've added Developer Response to some of them. 

This one (May 18) might deserve a short response. 

He has a point that new users who don't already know DailyGammon could use a short tutorial on getting started. 

I will draft something if you'd like, both as Developer Response, and also a Getting Started with DG App FAQ. 

![image](https://github.com/hape42/dailygammon/assets/121371957/3a4a6cd9-b78e-4aa1-8038-3172f304b594)

**Comment by kagsteiner** [comment]
Just logged out to see the initial screen you see as a new user. It's very simple and clear, so apart from being able to read (and maybe know the term FAQ) it should be quite obvious how to play...

**Comment by Smiles2go** [comment]
Agree, though might be able to dumb it down a bit more:

0. Register and confirm DailyGammon login. 
1. Login through this app.
2. Visit Game Lounge (menu button top-right)
3. Find "Beginners Tournament" and Sign Up. Others also. 
4. Wait for tournament to start - after enough others have signed up
5. Visit Top Page and tap tournament name to play

Could add more detail. Hardest to explain is delay after signup. How long to wait? How to tell when tournament starts?



0. (If no DailyGammon account) request one <here-link to req page> Be sure to enter correct email remember chose username/password and reply to confirmation email.

**Comment by hape42** [comment]
Ok, understood what you mean.

there are currently about 2500 downloads. there are 135 ratings with an average of 4.5 !

There are exactly eleven 1-star ratings, most of them like this:
"Not what I need" 

If the app isn't what the person is looking for, why don't they just ditch the app and keep looking? Why does he give it a 1 star rating? It's not the app's fault that he can't find what he's looking for. I've tried, but it's frustrating to talk to people like that.

I can add more information to the home page of the app, but I don't have time to keep track of all the reviews in the AppStore and respond accordingly.

If someone can write me the text for more information ready to copy, I'd be happy to add it to the app login page.

**Comment by Smiles2go** [comment]
I don't know what is best. 

Maybe a button on DD App login and About screen saying "Having trouble getting started?" That leads to a summary of how to get started. I'll finish a draft. 

Last two times I tried creating new DailyGammon login, I didn't get response with initial password. Possibly it considered address as disposable? 

Emailing them helped. 

Of course none of that is controlled by DG App.

**Comment by hape42** [comment]
maybe a button "what is DailyGammon about" and then a text pops up explaining what DailyGammon actually is and how it works

And maybe also the distinction between what the website does and what the app does or can't do
🤔

**Comment by Smiles2go** [comment]
I'll craft a draft to look at, though could be a day or few.

**Comment by kagsteiner** [comment]
> Note: two times I tried creating new DailyGammon login (including 3 days ago), I didn't get email response with initial password. Possibly it considered address as disposable?
> 
> Emailing them helped.
> 
> Of course none of that is controlled by DG App.

Interesting, I had the same experience, was very frustrating to join - tried two email addresses, one of gmail.com, one of mailbox.org, both without success; only after emailing them I got my account. Maybe even mention that?

**Comment by hape42** [comment]
> I'll craft a draft to look at, though could be a day or few.

I will already programme the button and the popup window. Then I only need to copy the text and can send a new version.

**Comment by Smiles2go** [comment]
Modal popup as scrollable text window should be good. Can avoid temptation to have clickable links.

**Comment by hape42** [comment]
![Simulator Screenshot - iPhone 15 Pro - 2024-07-12 at 10 50 56](https://github.com/user-attachments/assets/1ad58eea-d7cb-49e7-aa10-5d8e6cabb41d)

**Comment by Smiles2go** [comment]
Below is a suggested getting-started text. This is difficult to write, not knowing how much to say or where new users may have trouble. 

Improvements/ suggestions welcome. 

----------

Welcome to DailyGammon app, a way to play backgammon games and tournaments with real people from around the world, all at once. It is free and ad-free. 

Play is at your own pace with opponents who usually won’t be online at the same time as you. You can have dozens of games going all at once. 

This DailyGammon app is a client for the DailyGammon.com web service, started over 20 years ago. You can play directly from a web browser like Safari, but from iPhone or iPad, the web-play experience is challenging. 

This app helps by improving display, moves, and chat. And it can notify when you have games ready to play. 

Getting started is a bit messy. Here’s the process:

1. Register and confirm DailyGammon login.
2. Login through this app.
3. Visit Game Lounge (menu button top-right)
4. Find "Beginners Tournament" and Sign Up.
5. Wait for tournament to start, after enough others have signed up (a few hours to days)
6. Visit Top Page to see games where you can play, and tap opponent name to play. 

The hardest part to starting is getting your DailyGammon account setup. 

Be sure to enter a correct email and remember your chosen username. You should get email with your initial password within 24 hours. If you don’t receive it, email dghelp@gmail.com and include your requested username.

We hope that’s enough to get you started. There are more new-user hints at http://www.dailygammon.com/help/hints.html

Note that this app is NOT built by the creators of the DailyGammon service.

**Comment by kagsteiner** [comment]
Very well written! The only feedback I have is that word „messy“. Is it really messy? Maybe complicated? But I’m German, not that good at this. 
and should we mention that (according to my knowledge) this is the only place in the internet to play correspondence backgammon (which for me was the „unique selling point“ that brought me to the site and let me endure the appalling web UI.

your thoughts?

**Comment by Smiles2go** [comment]
"Complicated" instead of "messy" is fine. 

I tried to convey idea of correspondence / Postcard chess without saying "asynchronous" -- was something my dad did but he was born in 30s. That you can have dozens of games and opponent need not be online. Still, some may relate. Can add. 

Is DG really unique this way? So many game engines out there.

**Comment by hape42** [comment]
update is ready in testFlight for download

**Comment by Smiles2go** [comment]
Version 2.0 (683) from 13. July 2024 08:37:54

"Tell me more" button works, shows scrollable text. 

Maybe add that button also to About screen so can refer back after account is setup?  

It is most important with login screen where you put it.

**Comment by kagsteiner** [comment]
Hi also expected it on the about screen. Works nicely on the login screen on my phone. The about screen is already quite crowded so it’s probably best to leave it as is.

**Comment by Smiles2go** [comment]
Version 2.0 (683) from 13. July 2024 08:37:54

True About screen is crowded, but is mostly text looking for GitHub volunteers. Could reduce a sentence or two and add button on top.

**Comment by kagsteiner** [comment]
Good point. BTW @hape42 did we get any additional testers? Maybe we need to find another channel to give feedback, maybe github is too geeky for most? Not sure, maybe we (I volunteer if you wish) should write something in the chat, and just use Testflight for the testers? But that would be a messy mail flood that then needs to turned into github issues. Not nice.

**Comment by Smiles2go** [comment]
Getting started text on Login and About pages good now. With public release of 
2.0...

Final suggestion here would be Peter response to this guy: ![image](https://github.com/user-attachments/assets/52a5c1d0-5019-4a0f-be53-ea425c5c75d1)

Suggest something like:

You're correct DailyGammon is different. But once you get started, can be fun to play individuals or tournaments with opponents all over the world. You can play dozens of turn-based matches at a time, at your own pace. 

For help getting started please see Login or About screen:

"You don't know DailyGammon Yet?"

Tap: Tell Me More

**Comment by Smiles2go** [comment]
With public release of version 2.

Final suggestion here would be Peter response to the May 18th complaint. 

Suggest something like:

---

You're correct that DailyGammon is different - most setup free DailyGammon account to play. But once you get started, is fun to play individual games or tournaments with opponents all over the world. You can play dozens of turn-based matches at a time, at your own pace.

For help getting started please see Login or About screen:

"You don't know DailyGammon Yet?"

Tap: Tell Me More

---

**Comment by Smiles2go** [comment]
Cool. That might help. 

Don't need to respond to every comment. Is one more though that could help. 

Might also respond to the "Needs notifications" guy. 

Was DG v1.5 per release notes in App Store when Notifications were added. 

...

Thanks for suggesting notifications. You were not alone. This feature was added in version 1.5.

With notifications enabled, you can see if games are waiting for your next move before opening the app. 

...
